### DISCUSSION TITLE
SQL Error (1105): target: vc-audit.-.primary: vttablet: rpc error: code = Unavailable desc = error reading from server: EOF

### DISCUSSION THREAD

**Discussion by visualdatasolutionsau** [planetscale/discussion#537] [Bug Reports]
Hi there

Getting the above error when running a SELECT query on the database. Have been able to migrate data across using INSERT statements, so need some guidance on the error.

**Comment by jgreet** [comment]
Could you [open a ticket](https://support.planetscale.com/hc/en-us/requests/new) with the organization, database and branch name where you saw the errors as well as the approximate time? Assuming it's not just a network issue it could be that a component in the database cluster ran out of memory or restarted for some other reason, but we'd need to look at the logs to be sure. Thanks!

**Reply by visualdatasolutionsau** [reply]
Thanks - submitted.
